.github/workflows/3-test-docker.yml

```diff
@@ -54,7 +51,8 @@ jobs:
       # yamllint disable rule:indentation
       - name: 🤞 Run Image tests
         run: |
-          ARMORY_VERSION=`armory --version | sed -r 's/\+/\./g'`
+          ARMORY_VERSION=`python -m armory.utils.version`
```

Comment:
Why can't you just do `python -m armory --version`?

Reply:
That was a WIP hack where I added an custom `if __main__` to version.py. It is not going into this PR when ready for review.

Reply:
Do you have an opinion on using `python -m armory.utils.get_docker_version` to factor out the postprocessing? Where get_docker_version.py is
```
# docs…
if __name__…
    print(to_docker_tag(get_version()))
```

Reply:
It would be nice to use the same entrypoint in `__main__`, but I like having a `get_docker_version` function.

Also nice not to have:
```
/home/anaconda/anaconda/conda_envs/david.slater/armory39/lib/python3.9/runpy.py:127: RuntimeWarning: 'armory.utils.version' found in sys.modules after import of package 'armory.utils', but prior to execution of 'armory.utils.version'; this may result in unpredictable behaviour
  warn(RuntimeWarning(msg))
```

It would only be a couple of lines in main to add the docker version using the get_docker_version function.
https://github.com/twosixlabs/armory/blob/develop/armory/__main__.py#L747-L763

Reply:
This is now implemented as `armory --show-docker-version-tag` where the long, descriptive name is intentional. I am not documenting this in `Usage`